### DISCUSSION TITLE
Dezoomify take infinite time loading tiles in  generic dezoomer

### DISCUSSION THREAD

**Discussion by leal89** [lovasoa/dezoomify#683] [General]
hello , i used dezoomify for images from google arts and its ok, the dezoomer works fine , it takes long but at least 1 hour it end  fine, but now im tring to dezoom this image https://cdn.gerhard-richter.com/images/xxlarge/15477.jpg, its a link image from the website and i choose generic dezoomer option but this time really it takes infinite time loading tiles , after 30 min i still at 0 percent with micro visible tiles loading
can you help me
thanks

**Comment by MrChrisWin** [comment]
1. This is the wrong place to put this question. For the future, please ask in Issues
2. You cannot use dezoomer in this case. For deezomer to work, the site has to offer hi-res tiled image wrapped in some sort of panoramic technology. However on here the site only uploaded low-resolution image for the artwork. Also the painting was sold in 1996 on Christies.  Think back in 1996, digital camera technology was in its infancy. There is a very high probability that there's no high-resolution image available for the painting.
